app/src/main/res/layout/fragment_exposures.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <import type="org.covidwatch.android.ui.ItemBindings" />
+
+        <import type="android.view.View" />
+
+        <variable
+            name="viewModel"
+            type="org.covidwatch.android.ui.exposures.ExposuresViewModel" />
+    </data>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        tools:context="org.covidwatch.android.ui.exposurenotification.ExposureNotificationFragment">
+
+        <ImageView
+            android:id="@+id/btn_close"
+            style="@style/ImageView.BtnClose"
+            android:padding="@dimen/padding_content"
+            tools:ignore="ContentDescription" />
+
+        <TextView
+            style="@style/TextView.ScreenTitle"
+            android:paddingStart="@dimen/padding_content"
+            android:paddingEnd="@dimen/padding_content"
+            android:text="@string/title_exposures" />
+
+        <Switch
+            android:id="@+id/enable_exposure_notification"
+            style="@style/Switch.EnableExposureNotification"
+            android:checked="@{viewModel.exposureNotificationEnabled}"
+            android:paddingStart="@dimen/padding_content"
+            android:paddingEnd="@dimen/padding_content" />
```

Comment:
use `paddingHorizontal`

Reply:
This is very strange thing but you are probably right.

Do you know that this property was introduced only in Oreo? 
https://developer.android.com/about/versions/oreo/android-8.0

When I tried to add this property in the styles file i got a warning that you cannot use it as our API is `21` and even future `23` doesn't help. So I thought xml lint just doesn't work properly in the layouts files and used `android:paddingStart`, `android:paddingEnd`

But I double checked today and found this:
https://www.reddit.com/r/androiddev/comments/e2crzv/is_it_true_that_i_cant_use_layout/

AAPT converts for you `paddingHorizontal` into `android:paddingStart`, `android:paddingEnd` at build time.

https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/tools/aapt2/#version-2_16